frontend/src/metabase/reference/metrics/MetricDetail.jsx

```diff
@@ -56,208 +56,179 @@ const mapDispatchToProps = {
   // The state and callbacks are received via props
   ..._.omit(actions, "startEditing", "endEditing"),
 
+  onUpdate: actions.rUpdateMetricDetail,
   onChangeLocation: push,
 };
 
-const validate = (values, props) =>
+const validate = values =>
   !values.revision_message
```

Comment:
I'm getting a full app crash on this page every time because entity is undefined 🤔 

```
Uncaught TypeError: Cannot read properties of undefined (reading 'revision_message')
```

Reply:
nevermind! I just pulled again and it's good now 👍